### ISSUE TITLE
Question about finding default authenticated role

### ISSUE THREAD

**Issue by rndexe** [kucherenko/strapi-plugin-passwordless#7]
Hi,

I've been facing a weird intermittent issue where sometimes the newly created user get assigned a Public role instead of Authenticated role despite never changing the setting in User-Permissions.
I haven't been able to reliably reproduce it but I suspect it is this line

https://github.com/kucherenko/strapi-plugin-passwordless/blob/750f4f50b660b626b3cd6d739305d635d40cc2cc/server/services/passwordless.js#L51 

Is this syntax supported in v4? Should the params be an object as given in the docs at 

https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/query-engine/single-operations.html#findone
```
findOne( {  
    where : { type : userSettings.default_role }  
})
```

Please let me know if this is intended or am I missing something. Thank you.

**Comment by kucherenko** [comment]
thank you for the report, will fix it

**Comment by kucherenko** [comment]
The issue has been fixed at strapi-plugin-passwordless@4.0.8
